blockbase/inc/wpcom.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * WordPress.com-specific functions and definitions.
+ *
+ * This file is centrally included from `wp-content/mu-plugins/wpcom-theme-compat.php`.
+ *
+ * @package Blockbase
+ */
+
+
+/**
+ * Enqueue our WP.com styles for front-end.
+ * Loads after theme styles so we can add overrides.
+ */
+function blockbase_wpcom_scripts() {
+	wp_enqueue_style( 'blockbase-wpcom-style', get_stylesheet_directory_uri() . '/inc/wpcom-style.css', array( 'blockbase-styles' ) );
```

Comment:
`get_stylesheet_directory_url` wasn't working for me, I had to change that to `get_template_directory` to make it work (I think because it's a parent theme?  I dunno, those two confuse me...).  And the dependency for blockbase is `blockbase-ponyfill` rather than `blockbase-styles`.  Thus:
```
wp_enqueue_style( 'blockbase-wpcom-style', get_template_directory_uri() . '/inc/wpcom-style.css', array( 'blockbase-ponyfill' ) );
```